.github/workflows/updateProtectedBranch.yml

```diff
@@ -1,140 +0,0 @@
-# This is a utility workflow to create, approve, and merge a pull request against a protected branch.
-name: Update Protected Branch
-
-on:
-  workflow_dispatch:
-    inputs:
-      TARGET_BRANCH:
-        description: The target branch to update. This becomes the base branch of the pull request.
-        required: true
-      SOURCE_BRANCH:
-        description: If updating main, you must also provide a head branch to update main with.
-        required: false
-        default: ''
-
-jobs:
-  updateBranch:
-    if: github.actor == 'OSBotify'
-    runs-on: ubuntu-latest
-    steps:
-      - name: Validate target branch
-        if: ${{ !contains(fromJSON('["main", "staging", "production"]'), github.event.inputs.TARGET_BRANCH) }}
-        run: |
-          echo "Target branch must be one of ['main', 'staging', 'production']"
-          exit 1
-
-      # If updating main, SOURCE_BRANCH must not be empty
-      - name: Validate source branch
-        if: github.event.inputs.TARGET_BRANCH == 'main' && github.event.inputs.SOURCE_BRANCH == ''
-        run: |
-          echo "Cannot update main branch without specifying a source branch"
-          exit 1
-
-      # If updating staging, the source branch will always be main
-      # If updating production, the source branch will always be staging
-      - name: Set source branch
-        run: |
-          if [[ ${{ github.event.inputs.TARGET_BRANCH }} == 'staging' ]]; then
-            echo "SOURCE_BRANCH=main" >> "$GITHUB_ENV"
-          elif [[ ${{ github.event.inputs.TARGET_BRANCH }} == 'production' ]]; then
-            echo "SOURCE_BRANCH=staging" >> "$GITHUB_ENV"
-          else
-            echo "SOURCE_BRANCH=${{ github.event.inputs.SOURCE_BRANCH }}" >> "$GITHUB_ENV"
-          fi
-
-      # Version: 2.3.4
-      - uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f
```

Comment:
Reviewer note: This checkout is no longer needed because, unlike the workflow, the action runs on the same machine that triggers it. Since the action is loaded from a local URL, we know for sure that if we're in the action we will have already ran a checkout and will have the repo cloned on the runner.